src/pages/view-test-result/cat-adi-part2/components/test-details/__tests__/test-details.cat-adi-part2.spec.ts

```diff
@@ -0,0 +1,75 @@
+import { ComponentFixture, async, TestBed } from '@angular/core/testing';
+import { IonicModule, Config } from 'ionic-angular';
+import { ConfigMock } from 'ionic-mocks';
+import { MockComponent } from 'ng-mocks';
+import { configureTestSuite } from 'ng-bullet';
+import { DataRowComponent } from '../../../../../../components/common/data-row/data-row';
+import { DataRowCustomComponent } from '../../../../../../components/common/data-row-custom/data-row-custom';
+import { CandidateDetails, TestDetailsCardCatADI2Component } from '../test-details.cat-adi-part2';
+import { it } from '@angular/cli/lib/ast-tools/spec-utils';
+
+describe('TestDetailsCardCatADI2Component', () => {
+  let fixture: ComponentFixture<TestDetailsCardCatADI2Component>;
+  let component: TestDetailsCardCatADI2Component;
+
+  configureTestSuite(() => {
+    TestBed.configureTestingModule({
+      declarations: [
+        TestDetailsCardCatADI2Component,
+        MockComponent(DataRowComponent),
+        MockComponent(DataRowCustomComponent),
+      ],
+      imports: [
+        IonicModule,
+      ],
+      providers: [
+        { provide: Config, useFactory: () => ConfigMock.instance() },
+      ],
+    });
+  });
+
+  beforeEach(async(() => {
+    fixture = TestBed.createComponent(TestDetailsCardCatADI2Component);
+    component = fixture.componentInstance;
+  }));
+
+  describe('Class', () => {
+    // Unit tests for the components TypeScript class
```

Comment:
is this comment necessary?

Reply:
probably not, already existed just copied & amended the test to be ADI specific